lib/datasource/nuget/__snapshots__/index.spec.ts.snap

```diff
@@ -323,6 +323,7 @@ Object {
       "version": "2.5.10.11092",
     },
     Object {
+      "isDeprecated": true,
```

Comment:
The existing test fixtures contained several unlisted/deprecated packages, so I deliberately did not write additional tests.